server/game/cards/02.3-EDS/TheMayorsOffice.js

```diff
@@ -0,0 +1,63 @@
+const DeedCard = require('../../deedcard.js');
+
+class TheMayorsOffice extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Noon: Modify Influence',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                cardCondition: {location: 'play area', condition: card => (card.gamelocation === this.gamelocation || card.isAdjacent(this.gamelocation))},
+                cardType: 'dude',
+                gameAction: ['increaseInfluence', 'decreaseInfluence']
+            },
+            handler: context => {
+                this.abilityContext = context;
+                this.game.promptWithMenu(context.player, this, {
+                    activePrompt: {
+                        menuTitle: 'Raise or lower their influence by one?',
```

Comment:
I would use the card name in the title and make it a little shorter. The exact change is in the option names.
```suggestion
                        menuTitle: `How to change ${context.target}'s influence?',
```
Or something like that.